src/main/java/io/kestra/plugin/fs/vfs/Downloads.java

```diff
@@ -49,6 +49,12 @@ public abstract class Downloads extends AbstractVfsTask implements RunnableTask<
     @PluginProperty(dynamic = true)
     private String regExp;
 
+    @Schema(
+        title = "List file recursively"
+    )
+    @PluginProperty(dynamic = true)
+    private boolean recursive;
```

Comment:
```suggestion
    @PluginProperty
    @Builder.Default
    private Boolean recursive = false;
```

Boolean cannot be dynamic and we usually set a default and `@Builder.Default` so it's clearly documented.

Reply:
Same for all